integration_test/tests/static_page_test.dart

```diff
@@ -0,0 +1,93 @@
+
+
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import '../commons/test_util.dart';
+import '../pages/onBoarding_page.dart';
+import '../pages/setting_page.dart';
+
+void main() async{
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group("static page test", (){
+    testWidgets("Release notes", (tester) async{
+
+      await initAppAutonomy(tester);
+      await launchAutonomy(tester);
+      await onboardingSteps(tester);
+      await addDelay(3000);
+
+      final releaseNotes = find.text("Release notes");
```

Comment:
Please move the element locator to the corresponding page (to the corresponding "_page") to easy for maintaining.